eclass/distutils-r1.eclass

```diff
@@ -1386,9 +1390,11 @@ distutils_pep517_install() {
 			)
 			;;
 		meson-python)
+			local mesonargs=()
```

Comment:
Also add `local` for all the other variables set by `setup_meson_src_configure`.